ui/__tests__/components/disclaimer.test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import Disclaimer from '../../components/disclaimer';
+import { email } from '../../components/contact-email';
+
+describe('<Disclaimer />', () => {
+  describe('returns the right contents', () => {
+    it('has the right number of div elements', () => {
```

Comment:
I think tests that are tied too closely to the markup are a bit fragile (since they're easy to break). What do you think about testing for some of the content within divs instead? Something like:
```js
expect(result.text()).toMatch(/some string/);
```

Reply:
Good point.